save/action.yml

```diff
@@ -11,9 +11,12 @@ inputs:
   upload-chunk-size:
     description: 'The chunk size used to split up large files during upload, in bytes'
     required: false
+  matched-key:
```

Comment:
This is not correct. isn't it? Both primary key and matched key will use same value as State constants. No?
https://github.com/actions/cache/blob/be72d0432d4f55dc4892a372e19e129b600b26b5/src/constants.ts#L12

Reply:
Good point, we will use different names so that the inputs/states/outputs don't coincide.